src/main/resources/common-services/CDAP/4.0.0/configuration/cdap-site.xml

```diff
@@ -497,6 +522,20 @@
   </property>
 
   <property>
+    <name>data.tx.grace.period</name>
+    <value>86400</value>
+    <description>
+      Time in seconds used to pad transaction maximum lifetime while pruning
+    </description>
+    <final>true</final>
+    <value-attributes>
+      <type>int</type>
+      <minimum>1</minimum>
+      <overridable>false</overridable>
```

Comment:
Add a `unit` tag and Ambari will allow auto-conversion of units (user can enter 1d and it'll convert to 86400)